cron/healthToMongo.py

```diff
@@ -6,45 +6,20 @@
 # Copyright (c) 2014 Mozilla Corporation
 
 
-import logging
 import requests
 import sys
 from datetime import datetime
 from configlib import getConfig, OptionParser
-from logging.handlers import SysLogHandler
 from pymongo import MongoClient
 
+from mozdef_util.utilities.logger import logger, initLogger
 from mozdef_util.utilities.toUTC import toUTC
 from mozdef_util.elasticsearch_client import ElasticsearchClient
 from mozdef_util.query_models import SearchQuery, TermMatch
 
 logger = logging.getLogger(sys.argv[0])
```

Comment:
We need to delete this line now that we're using mozdef_util/utilities/logger

Reply:
will do!

Reply:
done!